packages/backend/src/controllers/savedChartController.ts

```diff
@@ -47,6 +48,7 @@ export class SavedChartController extends Controller {
         body: {
             dashboardFilters?: any; // DashboardFilters; temp disable validation
             invalidateCache?: boolean;
+            dashboardSorts?: SortField[];
```

Comment:
in the context of this `API request` these filters and sorts are applied to a single chart. 

I didn't want to say `override` , and be more generic, in case we change the logic and decide to `append` like we do in filters, but that is just a personal preference,  I can change if you prefer.